specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -862,6 +862,107 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/allowedVmSizesForResize": {
```

Comment:
Please define these two new operations under "paths" section, not outside. That's why you are getting Swagger SemanticValidation failed errors.

Reply:
Fixed